templates/system/page.html.twig

```diff
@@ -54,10 +54,10 @@
 #}
 {%
   set content_classes = [
-    has_sidebar_first and page.sidebar_second ? 'col-sm-6',
-    has_sidebar_first and not has_sidebar_second ? 'col-sm-8',
-    not has_sidebar_first and has_sidebar_second ? 'col-sm-8',
-    not has_sidebar_first and not has_sidebar_second ? 'col-sm-12',
+    has_sidebar_first and page.sidebar_second ? 'col-md-6',
```

Comment:
I wonder why we are using page.sidebar_second instead of has_sidebar_second here.

Reply:
I believe @stephen-cox set this up originally

Reply:
Yeah, possibly.  Since you didn't change it, let's leave it as it is.  *has_sidebar_second* and *page.sidebar_second* could have different values which is why I am worried this can lead to bugs.